### PR TITLE
Update VCR cassettes to work correctly with VCR3+

### PR DESCRIPTION
**Pull request by cfcosta**
The cassette's format was changed, with the removal of basic auth from
the URL, being moved to its own Authorization header field.

This is part of the effort to make the gem support Ruby 2.4.

References:
https://github.com/ManageIQ/manageiq-providers-hawkular/pull/17
https://github.com/ManageIQ/manageiq/pull/14270

### CODE REVIEW

**Comment by cfcosta** [comment]
@Jiri-Kremser @cben @jrafanie can you take a look at this, please?

**Comment by jrafanie** [comment]
Nice @cfcosta, does this branch work for you with ruby 2.4?  If so, we can then add it to the `.travis.yml` test matrix in a followup... or here.

Awesome ❤️

**Review by jkremser — COMMENTED** [review]
lgtm, could you pls add the 2.4(.1)? to the `.travis.yml` to see if travis also likes it? (as @jrafanie asked)

Are the tests for operations also passing? I am asking because these 22 were failing for me and I didn't have the instance that wasn't running in the container at that moment.

**Comment by jrafanie** [comment]
@cfcosta hmm, can you try with `webmock ~> 2.3.1` to see if this passes on ruby 2.4?  I wonder if we're hitting another upgrade issue when we go to webmock 3+.

**Comment by josejulio** [comment]
The current error could be a 'bug' from the websocket library (some tests fail from time to time) or some bug not related to this one (I have seen that fail on other PRs). It could be fixed by #205. For now, I've restarted the test to see if it passes.

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ sudo: required
 
 language: ruby
 rvm:
-  - 2.2
-  - 2.3.0
+  - 2.3.3
```
Comment (josejulio) [inline .travis.yml]: Are we ready to drop support for 2.2.?
I think ManageIQ supports 2.2.2 at minimum
http://manageiq.org/docs/guides/developer_setup#install-ruby-and-bundler

Comment (jrafanie) [reply .travis.yml]: I would keep 2.2 for now... we have intentions to drop 2.2 in manageiq soon but we're not there yet.

See: http://talk.manageiq.org/t/adding-support-for-ruby-2-4-and-dropping-2-2/2305

Comment (cfcosta) [reply .travis.yml]: Done.

**Review by jrafanie — APPROVED** [review]
:shipit:

**Comment by Jiri-Kremser** [comment]
If @josejulio confirms, it can be merged

**Comment by jkremser** [comment]
If @josejulio confirms, it can be merged

**Review by josejulio — APPROVED** [review]

**Pull request merged**